src/Plugin/DataType/LoggerEntry.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\DataType\LoggerEntry.
+ */
+
+namespace Drupal\rules\Plugin\DataType;
+
+use Drupal\Core\TypedData\Plugin\DataType\Map;
+
+/**
+ * The logger entry data type.
+ *
+ * @see: https://www.drupal.org/node/2625238
```

Comment:
the "@see" annotations should not have a colon after it, see https://www.drupal.org/coding-standards/docs#see